static/app/components/asyncComponent.tsx

```diff
@@ -80,9 +80,7 @@ class AsyncComponent<
     if (props.routes && props.routes) {
       metric.mark({name: `async-component-${getRouteStringFromRoutes(props.routes)}`});
     }
-  }
 
-  UNSAFE_componentWillMount() {
```

Comment:
This should probably be in `didMount` no? It would match the other PR calling `Super.componentDidMount` on this.

Reply:
yeah I had it as `didMount` and it was failing tests, so tried in the constructor, since that's most similar to willMount